roles/pulp-webserver/tasks/apache.yml

```diff
@@ -22,6 +22,11 @@
         name: "{{ pulp_webserver_apache_package }}"
         state: present
 
+    - name: Install mod_ssl
+      package:
+        name: "mod_ssl"
+        state: present
```

Comment:
WRT to Fedora/EL:
1. Let's set `pulp_webserver_apache_package` to `mod_ssl`. Beecause `mod_ssl` depends on the existing package, `httpd`, and there is no variable for this role to install apache without SSL support.

WRT Debian:
1. This package does not exist at all.
2. When we install the package "apache2" above, ssl support is already installed. The deficiency is that it is not enabled.
You need to resolve this on Debian by enabling 1 to 3 of the following distro-provided snippets:
```
root@40ff32e7c273:/etc/apache2# find /etc/apache2 | grep ssl
/etc/apache2/mods-available/ssl.conf
/etc/apache2/mods-available/ssl.load
/etc/apache2/sites-available/default-ssl.conf
```
Determine which of those 3 you need to enable, and then see the task in apache.yml, `name: Enable Apache vhost files and mod files` on how we symlink snippets to enable them.

Reminder: You can test/investigate this easily by running:
`docker run -it debian:10 /bin/bash`

Reply:
Thank you for this comment I will look into both of these aspects.